roles/host_done/tasks/main.yml

```diff
@@ -1,5 +1,9 @@
 ---
 
+- name: internal_host_started_processing_role
+  set_fact:
+    internal_host_started_processing_role: "host_done"
+
 - name: extract virt-what error code
```

Comment:
Seems like a bad task name that came in.

Reply:
Fixed.